### PR TITLE
DDF-2393:fixed branding usage in docs intro

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?

Updates use of ${branding} in Introductory section of documentation that should be ${ddf-branding}.

#### Who is reviewing it? 

@mcalcote @vinamartin
#### Choose 2 committers to review/merge the PR.

@coyotesqrl
@shaundmorris

#### How should this be tested?

docs render correctly. 

#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-2393

#### Screenshots (if appropriate)
n/a
#### Checklist:
- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

**Comment by coyotesqrl** [comment]
👍

**Comment by shaundmorris** [comment]
👍

**Comment by shaundmorris** [comment]
This is based on the new outline, so shouldn't need to be backported, correct?

**Comment by ricklarsen** [comment]
@shaundmorris Actually, I updated it in the new outline as well on this PR, so that we _could_ backport this, but I'm not 100% sure we need to.

File: distribution/docs/src/main/resources/_contents/introduction-contents.adoc [hunk 1]
```diff
@@ -13,27 +13,27 @@ ${ddf-branding} is:
 ${branding} is comprised of several modular applications, to be installed or uninstalled as needed.
 
 ${ddf-branding} Administrative Application::
-The administrative application enhances administrative capabilities when installing and managing ${branding}. It contains various services and interfaces that allow administrators more control over their systems.
+_${ddf-branding} version ${ddf.version}_ The administrative application enhances administrative capabilities when installing and managing ${branding}. It contains various services and interfaces that allow administrators more control over their systems.
 
 ${ddf-branding} Catalog Application::
-The ${branding} Catalog provides a framework for storing, searching, processing, and transforming information.
+_${ddf-branding} version ${ddf.version}_ The ${branding} Catalog provides a framework for storing, searching, processing, and transforming information.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/introduction-contents.adoc]: Should this second `${branding}` in this line be updated?

File: distribution/docs/src/main/resources/_contents/introduction-contents.adoc [hunk 2]
```diff
@@ -13,27 +13,27 @@ ${ddf-branding} is:
...
+_${ddf-branding} version ${ddf.version}_ The Security application provides authentication, authorization, and auditing services for the ${branding}.
 It is both a framework that developers and integrators can extend and a reference implementation that meets security requirements.
 
 ${ddf-branding} Solr Catalog Application::
-The Solr Catalog Provider (SCP) is an implementation of the `CatalogProvider` interface using http://lucene.apache.org/solr/[Apache Solr] as a data store.
+_${ddf-branding} version ${ddf.version}_ The Solr Catalog Provider (SCP) is an implementation of the `CatalogProvider` interface using http://lucene.apache.org/solr/[Apache Solr] as a data store.
 
 ${ddf-branding} Spatial Application::
-The ${ddf-branding} Spatial Application provides KML transformer and a KML network link endpoint that allows a user to generate a View-based KML Query Results Network Link.
+_${ddf-branding} version ${ddf.version}_ The ${ddf-branding} Spatial Application provides KML transformer and a KML network link endpoint that allows a user to generate a View-based KML Query Results Network Link.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/introduction-contents.adoc]: Take out or use consistently on other sections, I think.

File: distribution/docs/src/main/resources/_contents/introduction-contents.adoc [hunk 3]
```diff
@@ -13,27 +13,27 @@ ${ddf-branding} is:
...
 
 ${ddf-branding} Standard Search UI::
-The ${ddf-branding} Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources.
+_${ddf-branding} version ${ddf.version}_ The ${ddf-branding} Standard Search UI application allows a user to search for records in the local Catalog (provider) and federated sources.
```
Comment (coyotesqrl) [inline distribution/docs/src/main/resources/_contents/introduction-contents.adoc]: Same. Take out or use consistently.

File: distribution/docs/src/main/resources/_contents/introduction-contents.adoc (hunk 1 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/introduction-contents.adoc]: removed.

File: distribution/docs/src/main/resources/_contents/introduction-contents.adoc (hunk 3 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/introduction-contents.adoc]: removed.

File: distribution/docs/src/main/resources/_contents/introduction-contents.adoc (hunk 2 above)
Comment (ricklarsen) [inline distribution/docs/src/main/resources/_contents/introduction-contents.adoc]: removed.

**Comment by coyotesqrl** [comment]
The changes look good. 👍

**Comment by vinamartin** [comment]
LGTM

**Pull request merged**